ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -50,6 +51,10 @@ public static class NotificationSubPaths
 
         public static string EditManualTestResult(int? testResultId) => $"Edit/ManualTestResult/{testResultId}";
         public static string AddManualTestResult => $"Edit/ManualTestResult/New";
+
+        public static string EditSocialContextVenue(int? socialContextVenueId) => $"Edit/SocialContextVenue/{socialContextVenueId}";
+        public static string AddSocialContextVenue => "Edit/SocialContextVenue/New";
+        public static string EditSocialContextVenueSubPath => "Edit/SocialContextVenue/";

```

Comment:
What's this one used for? Curious, as it doesn't follow the rest of it really?

Reply:
This was needed in the tests - as we need to create the full path (without id) differently in different tests